lms/djangoapps/onboarding/forms.py

```diff
@@ -420,8 +416,7 @@ def save(self, request, commit=True):
             user_info_survey.start_month_year = None
             user_info_survey.hours_per_week = 0
             user_info_survey.role_in_org = None
-            selected_function_areas = ""
-            user_info_survey.user.extended_profile.save_user_function_areas(selected_function_areas)
+            user_info_survey.user.extended_profile.save_user_function_areas(list())
```

Comment:
Why are we passing this empty list here?

It is resulting in an error upon adding function related information:

File "/edx/app/edxapp/edx-platform/lms/djangoapps/onboarding/models.py", line 527, in get_user_selected_functions
for function_area in self.function_areas]
AttributeError: 'list' object has no attribute ‘choices'

Please look into this. To replicate the issue, go to:
http://local.philanthropyu.org:18000/myaccount/additional_information/

and add Department or Function related information.